### DISCUSSION TITLE
自定义ArrayCards.Addition不显示

### DISCUSSION THREAD

**Discussion by zhanghy7** [alibaba/formily#3260] [Q&A 2.x]
export function ArrayCardsAdd(props: any) {
  const { title } = props;

  return <ArrayCards.Addition {...props} title={title || '123'} method="push" defaultValue={[]} />
}

在schema中使用，新增按钮无法显示

**Comment by zFitness** [comment]
上个 codesanboxs 呗， 你这都没有说怎么用的

**Comment by zhanghy7** [comment]
看了源码，自定义的组件名称需要包含Addition字符串。。。否则不生效

**Reply by jiojio-jx** [reply]
代码能copy一下么，阅览下
